contrib/IECoreUSD/src/IECoreUSD/USDScene.cpp

```diff
@@ -2116,7 +2133,11 @@ void USDScene::childNames( SceneInterface::NameList &childNames ) const
 
 SceneInterfacePtr USDScene::child( const SceneInterface::Name &name, SceneInterface::MissingBehaviour missingBehaviour )
 {
-	pxr::UsdPrim childPrim = m_location->prim.GetChild( pxr::TfToken( name.string() ) );
+	pxr::UsdPrim childPrim;
+	if( pxr::TfIsValidIdentifier( name.string() ) )
```

Comment:
I guess `scene` is doing all kinds of not-following-its-args anyway, so leaving this to warn, is prob ok for now?

https://github.com/ImageEngine/cortex/blob/2deca9ad1cceeb4ba3c7c37650aaf222272a36b3/contrib/IECoreUSD/src/IECoreUSD/USDScene.cpp#L2193